src/interfaces/IInvestmentManager.sol

```diff
@@ -45,7 +45,7 @@ interface IInvestmentManager {
     ///         proceed with tranche token payouts in case their orders got fulfilled.
     /// @dev    The user asset amount required to fulfill the deposit request have to be locked,
     ///         even though the tranche token payout can only happen after epoch execution.
-    function requestDeposit(address vault, uint256 assets, address receiver, address owner, address souce)
+    function requestDeposit(address vault, uint256 assets, address receiver, address owner, address source)
```

Comment:
I think this is something we are interested in general.
I noticed that in the events ( such as  the Deposit one ) we emit who is the sender. 
Given that, I have decided that we can added anyways and use it.